rotate.c

```diff
@@ -300,6 +344,25 @@ int rotate_map(struct context *cnt, unsigned char *map)
         size = wh;
     }
 
+    switch (axis) {
+    case FLIP_TYPE_HORIZONTAL:
+        flip_inplace_horizontal(map,width, height);
+           if (cnt->imgs.type == VIDEO_PALETTE_YUV420P) {
```

Comment:
The indentation of the 'if' here looks wrong in the diff view (if it looks okay in your viewer it might be there's a tab somewhere...)